frontend/src/app/shared/components/dropdown/dropdown.component.html

```diff
@@ -5,13 +5,13 @@
                 {{selectedItems.length || 0}}
             </div>
         </div>
-        <div *ngIf="!selectedItems.length" class="dropdown-btn-title" >
+        <div *ngIf="selectedItems.length !== 1" class="dropdown-btn-title">
             {{getFullTitle()}}
         </div>
-        <div *ngIf="selectedItems.length" class="dropdown-btn-title-when-selected" >
+        <div *ngIf="selectedItems.length === 1" class="dropdown-btn-title-when-selected">
             {{getFullTitle()}}
         </div>
```

Comment:
We have 2 similar div with only difference in class name, may be it`s better to have 1, but add ngClass or className

Reply:
fixed